src/structures/GuildMember.js

```diff
@@ -210,6 +211,7 @@ class GuildMember extends Base {
    * @readonly
    */
   get kickable() {
+    if (!this.guild.me) throw new Error('GUILD_UNCACHED_ME');
```

Comment:
This check is unnecessary as manageable throws when `this.guild.me` is null. Same for bannable.

Reply:
That would throw `Cannot read property '*' of null` which isn't very helpful... hence why this PR throws the custom error message.